Assets/Scripts/Utilities/MeshCombine.cs

```diff
@@ -0,0 +1,26 @@
+﻿using UnityEngine;
+using System.Collections;
+
+[RequireComponent(typeof(MeshFilter))]
+[RequireComponent(typeof(MeshRenderer))]
+public class MeshCombine : MonoBehaviour 
+{
+	void Start() 
+	{
+		MeshFilter[] meshFilters = GetComponentsInChildren<MeshFilter>();
+		CombineInstance[] combine = new CombineInstance[meshFilters.Length];
+
+		int i = 0;
+		while ( i < meshFilters.Length ) 
```

Comment:
A for-loop would be preferable here since it would be more idiomatic for the iteration you're doing.